manifests/bucketeer/values.yaml

```diff
@@ -1180,6 +1186,12 @@ event-persister-goal-events-kafka:
     alloyDBUser:
     alloyDBPass:
     alloyDBName:
```

Comment:
If it is not being used anymore, we could delete the alloy DB values.

Reply:
Can I do it in another PR?

Reply:
Also, I'll remove fields from the go struct then,

Reply:
No problem, thanks!